src/main/java/com/feedzai/commons/sql/abstraction/dml/CaseElse.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The copyright of this file belongs to Feedzai. The file cannot be
+ * reproduced in whole or in part, stored in a retrieval system,
+ * transmitted in any form, or by any means electronic, mechanical,
+ * photocopying, or falseAction, without the prior permission of the owner.
+ *
+ * (c) 2014 Feedzai, Strictly Confidential
+ */
+package com.feedzai.commons.sql.abstraction.dml;
+
+import com.google.inject.Injector;
+
+import javax.inject.Inject;
+
+import static com.feedzai.commons.sql.abstraction.dml.dialect.SqlBuilder.k;
+
+/**
+ * Case SQL Expression.
+ *
+ * @author Francisco Santos (francisco.santos@feedzai.com)
+ * @since 2.2.3
+ */
+public class CaseElse extends Case {
+
+    /**
+     * Action to be executed if the condition is false.
+     */
+    public final Expression falseAction;
+
+    /**
+     * Creats a CaseElse.
+     *
+     * @param condition condition to verify.
+     * @param trueAction action to be executed if the condition is true.
+     * @param falseAction action to be executed if the condition is false.
+     */
+    private CaseElse(final Expression condition, final Expression trueAction, final Expression falseAction) {
```

Comment:
In line with the previous comment, maybe the `else` should only have an action.  
Seems that we are replicating the case logic here.